src/server/game/Handlers/NPCHandler.cpp

```diff
@@ -345,6 +345,18 @@ void WorldSession::HandleGossipHelloOpcode(WorldPackets::NPC::Hello& packet)
     unit->AI()->sGossipHello(_player);
 }
 
+void WorldSession::HandleCloseInteractionOpcode(WorldPackets::NPC::CloseInteraction& packet)
+{
+    TC_LOG_DEBUG("network", "WORLD: CMSG_CLOSE_INTERACTION");
+    Creature* unit = GetPlayer()->GetNPCIfCanInteractWith(packet.UnitGuid, UNIT_NPC_FLAG_NONE);
+    if (!unit)
+    {
+        TC_LOG_DEBUG("network", "WORLD: HandleCloseInteractionOpcode - %s not found, nothing to handle.", packet.UnitGuid.ToString().c_str());
+        return;
+    }
+    
```

Comment:
Do not remove yet, it's better to find out what is needed to be handled. Or I believe there is a special opcode table flag that marks opcode as not needed and ignore any errors.